### PR TITLE
Consistency edits

### PR DESCRIPTION
**Pull request by afloresOSI**
This is a start for the consistency edit of procedures.

### CODE REVIEW

**Review by VictoriaTouati-AVEVA — APPROVED** [review]

File: content/set-up/users/users-procedure.md [hunk 1]
```diff
@@ -4,31 +4,23 @@ uid: gpUsers
...
+1. In the **Identity Provider** field, select the appropriate identity provider for the user you are adding. You can switch to a different option later. For tenants with a single identity provider enabled, the field is disabled.
 
-1. Enter a first and last name for the user.  
+1. In the **Contact First Name** and **Contact Last Name** fields, enter a first and last name for the user.  
 
-1. Enter the contact email. The invitation to your tenant is sent to the user at this address. Ensure that it is correct so that the invitation is not sent to an unintended recipient.
+1. In the **Contact Email** field, enter the contact email. The invitation is sent to the user at this email address. Ensure that the email address is correct so that the invitation is not sent to an unintended recipient.
 
 1. Optional: Enable additional roles for the user. By default, the user is assigned the Tenant Member role which cannot be removed. Roles can be modified after the user is invited.
```
Comment (rmuolic) [inline content/set-up/users/users-procedure.md:19]:
```suggestion
1. (Optional) Enable additional roles for the user. By default, the user is assigned the Tenant Member role which cannot be removed. Roles can be modified after the user is invited.
```

File: content/add-organize-data/store-data/types/types-procedure.md [hunk 2]
```diff
@@ -4,56 +4,38 @@ uid: gpTypes
...
-   
-   - `Base Type` &ndash; (Optional) Select a previously created type on which the new type will be based. The new type inherits the properties of the base type. Inherited properties are read only and cannot be modified.
+ - **Id** &ndash; Enter the Id for the type.
+
+ - **Name** &ndash; (Optional) Enter a user-friendly name. If you do not enter a name, the **Id** is used as the name.
+
+ - **Description** &ndash; (Optional) Enter a user-friendly description of the type.
 
-1. Click  **Add Property** to add a property.
+ - **Base Type** &ndash; (Optional) To base the new type on an existing type, select the existing type from the dropdown. The new type inherits the properties of the base type. Inherited properties are read only and cannot be modified.
```
Comment (rmuolic) [inline content/add-organize-data/store-data/types/types-procedure.md:25]:
```suggestion
 - (Optional) **Base Type** &ndash; To base the new type on an existing type, select the existing type from the dropdown. The new type inherits the properties of the base type. Inherited properties are read only and cannot be modified.
```

**Review by rmuolic — APPROVED** [review]
Recommend to update instances of &ndash" to "-"

File: content/set-up/clients/authorization-code-client-procedure.md [hunk 3]
```diff
@@ -3,42 +3,37 @@ uid: gpAuthorizationCodeClient
...
 # Add an authorization code client
 
-Authorization code clients are used with customer web applications that use OCS as their backend. They provide a secure means of authenticating users of the website to view assets. See the following topics for more information about authorization code clients:
-
-- [Introduction to authorization code clients](xref:ccClients#authorization-code-client)
-- [Authorization code client PI Server counterpart](xref:ccClients#authorization-code-pi-server)
-- [Authorization code best practices](xref:ccClients#authorization-code-bp)
-
-## Procedure
+Authorization code clients provide a secure means of authenticating users to customer web applications that use OCS as their backend. Follow this procedure to add an authorization code client.
```
Comment (afloresOSI) [inline content/set-up/clients/authorization-code-client-procedure.md:6]:
```suggestion
Authorization code clients provide a secure means of authenticating users to customer web applications that use OCS as their backend. To add an authorization code client, follow these steps:
```

File: content/set-up/clients/authorization-code-client-procedure.md [hunk 4]
```diff
@@ -3,42 +3,37 @@ uid: gpAuthorizationCodeClient
...
 
 1. In the left pane, select **Security** > **Clients**.
 
-1. From the **Client Type** dropdown list, select **Authorization Code**.
+1. In the **Client Type** field, select **Authorization Code**.
 
 1. In the toolbar, click **Add Client**.
```
Comment (afloresOSI) [inline content/set-up/clients/authorization-code-client-procedure.md:12]:
```suggestion
1. In the toolbar, select **Add Client**.
```

File: content/add-organize-data/store-data/types/types-procedure.md [hunk 5]
```diff
@@ -4,56 +4,38 @@ uid: gpTypes
...
+To add a stream, follow these steps:
 
 1. In the left pane, select **Data Management** > **Sequential Data Store**.
 
-1. Click the **Change Namespace** button in the upper-right toolbar, then select the namespace in which you want to create the stream.
-
-1. In the left corner, click **Streams** > **Types**.
+1. From the **Streams** dropdown, select **Types**.
 
 1. Click **Add Type**.
```
Comment (afloresOSI) [inline content/add-organize-data/store-data/types/types-procedure.md:15]:
```suggestion
1. Select **Add Type**.
```

Comment (afloresOSI) [inline content/add-organize-data/store-data/types/types-procedure.md]:
```suggestion
1. Select **Add Type**.
```

File: content/set-up/clients/authorization-code-client-procedure.md [hunk 6]
```diff
@@ -3,42 +3,37 @@ uid: gpAuthorizationCodeClient
...
-   This can be used to designate other URLs from which the application is allowed to make requests against OCS. For example, this may be necessary if the user authenticates from an application running at one URL but the result of the authentication is sent to an application running at a different URL.
+   Designate other URLs from which the application is allowed to make requests against OCS. For example, this may be necessary if the user authenticates from an application running at one URL but the result of the authentication is sent to an application running at a different URL.
    
-1. (Optional) Set the **Token Lifetime**.  
+1. (Optional) In the **Token Lifetime** field, enter the length of time in seconds that the access token functions before it expires.  
 
-   Token Lifetime represents the length of time (in seconds) the access token is valid before it expires. The default, 3600 seconds (one hour), is the maximum length of time. The minimum value is 60 seconds.
+   The default, 3600 seconds (one hour), is the maximum length of time. The minimum value is 60 seconds.
 
 1. Click **Add**.
```
Comment (afloresOSI) [inline content/set-up/clients/authorization-code-client-procedure.md:34]:
```suggestion
1. Select **Add**.  
```

File: content/set-up/clients/client-credentials-client-procedure.md [hunk 7]
```diff
@@ -4,49 +4,45 @@ uid: gpClientCredentialsClient
...
-
-## Procedure
+Client-credentials clients are used for server-to-server communication where no user interaction is required. Follow this procedure to add a client-credentials client.
 
 1. In the left pane, select **Security** > **Clients**.
 
-1. From the **Client Type** dropdown list, select **Client-Credentials**. (This is the option that is displayed, by default.)
+1. In the **Client Type** field, select **Client-Credentials**. This option is displayed by default.
 
 1. In the toolbar, click **Add Client**.
```
Comment (afloresOSI) [inline content/set-up/clients/client-credentials-client-procedure.md:13]:
```suggestion
1. In the toolbar, select **Add Client**.
```

File: content/set-up/clients/authorization-code-client-procedure.md [hunk 8]
```diff
@@ -3,42 +3,37 @@ uid: gpAuthorizationCodeClient
...
+1. (Optional) In the **Token Lifetime** field, enter the length of time in seconds that the access token functions before it expires.  
 
-   Token Lifetime represents the length of time (in seconds) the access token is valid before it expires. The default, 3600 seconds (one hour), is the maximum length of time. The minimum value is 60 seconds.
+   The default, 3600 seconds (one hour), is the maximum length of time. The minimum value is 60 seconds.
 
-1. Click **Add**.  
+1. Select **Add**.  
 
-   The `Client Successfully Created` window displays the client Id for the client. The application must specify this client Id when it makes an authentication request. After you close this window, you can also retrieve the client Id from the list of clients.
+   The `Client Successfully Created` window displays the client Id for the client. The application must specify this client Id when it makes an authentication request. Copy the client Id by clicking the **Copy** icon. After you close this window, you can also retrieve the client Id from the list of clients.
```
Comment (afloresOSI) [inline content/set-up/clients/authorization-code-client-procedure.md:36]:
```suggestion
   The `Client Successfully Created` window displays the client Id for the client. The application must specify this client Id when it makes an authentication request. To copy the client Id, select the **Copy** icon. After you close this window, you can also retrieve the client Id from the list of clients.
```

File: content/set-up/clients/authorization-code-client-procedure.md [hunk 9]
```diff
@@ -3,42 +3,37 @@ uid: gpAuthorizationCodeClient
...
 
-1. In the toolbar, click **Add Client**.
+  The `Add Client` window appears.
 
-1. Enter a name for the client.  
-   Use this name to identify the device or application that will use this client.
+1. In the **Name** field, enter a name to identify the device or application that will use this client. 
 
-1. Enter the URLs in the **Allowed Redirect URL(s)** text box, and click **+** to add it to the list.
+1. In the **Allowed Redirect URL(s)** box, enter a URL and click **+** to add it to the list.
```
Comment (afloresOSI) [inline content/set-up/clients/authorization-code-client-procedure.md:18]:
```suggestion
1. In the **Allowed Redirect URL(s)** box, enter a URL and select **+** to add it to the list.
```

Comment (afloresOSI) [inline content/set-up/clients/authorization-code-client-procedure.md]:
```suggestion
1. In the **Allowed Redirect URL(s)** box, enter a URL and select **+** to add it to the list.
```

File: content/set-up/clients/authorization-code-client-procedure.md [hunk 10]
```diff
@@ -3,42 +3,37 @@ uid: gpAuthorizationCodeClient
...
 
    The application specifies one of the URLs in this list when it authenticates against OCS, and the OCS identity server returns the results of the authentication to this URL.
 
-1. (Optional) Enter URLs in the **Allowed Logout Redirect URL(s)** text box, and click **+** to add it to the list. 
+1. (Optional) In the **Allowed Logout Redirect URL(s)** box, enter a URL and click **+** to add it to the list.
```
Comment (afloresOSI) [inline content/set-up/clients/authorization-code-client-procedure.md:22]:
```suggestion
1. (Optional) In the **Allowed Logout Redirect URL(s)** box, enter a URL and select **+** to add it to the list. 
```

File: content/set-up/clients/authorization-code-client-procedure.md [hunk 11]
```diff
@@ -3,42 +3,37 @@ uid: gpAuthorizationCodeClient
...
    The application specifies one of the URLs in this list when it authenticates against OCS, and the OCS identity server returns the results of the authentication to this URL.
 
-1. (Optional) Enter URLs in the **Allowed Logout Redirect URL(s)** text box, and click **+** to add it to the list. 
+1. (Optional) In the **Allowed Logout Redirect URL(s)** box, enter a URL and click **+** to add it to the list. 
 
-   This is similar to the **Allowed Redirect URL(s)**. The application specifies a URL from this list when it logs out of OCS, and the OCS identity server sends the user to this URL after a successful log out.
+   The application specifies a URL from this list when it logs out of OCS, and the OCS identity server sends the user to this URL after a successful log out.
 
-1. (Optional) Enter URLs in the **Allowed CORS Origin(s)** text box, and click **+** to add it to the list.  
+1. (Optional) In the **Allowed CORS Origin(s)** box, enter a URL and click **+** to add it to the list.
```
Comment (afloresOSI) [inline content/set-up/clients/authorization-code-client-procedure.md:26]:
```suggestion
1. (Optional) In the **Allowed CORS Origin(s)** box, enter a URL and select **+** to add it to the list.  
```

File: content/set-up/clients/hybrid-client-procedure.md [hunk 12]
```diff
@@ -4,41 +4,53 @@ uid: gpHybridClient
...
 1. In the toolbar, click **Add Client**.
 
-1. Enter a name for the client.
+  The `Add Client` window appears.
+
+1. In the **Name** field, enter a name to identify the application that will use this client.
+
+1. (Optional) Select the **Allow Refresh Token** checkbox if the application uses refresh tokens to keep users logged in to the portal.
+
+1. In the **Allowed Redirect URL(s)** box, enter a URL and click **+** to add it to the list.
```
Comment (afloresOSI) [inline content/set-up/clients/hybrid-client-procedure.md:21]:
```suggestion
1. In the **Allowed Redirect URL(s)** box, enter a URL and select **+** to add it to the list.
```

File: content/set-up/clients/hybrid-client-procedure.md [hunk 13]
```diff
@@ -4,41 +4,53 @@ uid: gpHybridClient
...
+
+   The application specifies one of the URLs in this list when it authenticates against OCS, and the OCS identity server returns the results of the authentication to this URL.
+
+1. (Optional) In the **Allowed Logout Redirect URL(s)** box, enter a URL and click **+** to add it to the list.
```
Comment (afloresOSI) [inline content/set-up/clients/hybrid-client-procedure.md:25]:
```suggestion
1. (Optional) In the **Allowed Logout Redirect URL(s)** box, enter a URL and select **+** to add it to the list. 
```

**Pull request merged**